addons/mail/static/src/models/message_view/message_view.js

```diff
@@ -148,6 +155,9 @@ registerModel({
             isCausal: true,
             readonly: true,
         }),
+        authorOpenChatText: attr({
```

Comment:
I don't think having 'author' in the name is relevant here. It confused me at least.

Reply:
It's important to know it's in the "author" part of the message view that the text is shown.
I'll just rename the field to `authorTitleText`.

Reply:
How about `authorAvatarTitleText` then? What confuses me about `author` is that you can't guess exactly what it is in the interface without reading the code.

Apart from that, `TitleText` feels much more meaningful to me, good improvement 👍️